GWT.gitignore

```diff
@@ -14,3 +14,15 @@ gwt-unitCache/
 # more caches and things from deploy #
 war/WEB-INF/deploy/
 war/WEB-INF/classes/
+
+#compilation logs
+.gwt/
+
+#caching for already compiled files
+gwt-unitCache/
```

Comment:
Note this was a duplicate entry. c.f https://github.com/github/gitignore/pull/2741